stable/wordpress/README.md

```diff
@@ -123,11 +124,11 @@ The following repo contains the recommended production settings for wordpress ca
 To horizontally scale this chart, first download the [values-production.yaml](values-production.yaml) file to your local folder, then:
 
 ```console
-$ 
+$
```

Comment:
Can you remove this sign that does not add value?

Reply:
Done.